pennylane/math/quantum.py

```diff
@@ -247,31 +247,40 @@ def reduce_dm(density_matrix, indices, check_state=False, c_dtype="complex128"):
     return _permute_dense_matrix(density_matrix, sorted(indices), indices, batch_dim)
 
 
-def _batched_partial_trace(density_matrix, indices):
+def partial_trace(matrix, indices, c_dtype="complex128"):
     """Compute the reduced density matrix by tracing out the provided indices.
 
     Args:
-        density_matrix (tensor_like): 3D density matrix tensor. This tensor should be of size
-            ``(batch_dim, 2**N, 2**N)``, for some integer number of wires``N``.
+        matrix (tensor_like): 2D or 3D density matrix tensor. For a 2D tensor, the size is assumed to be
+            ``(2**n, 2**n)``, for some integer number of wires``n``.
+            ``(2**n, 2**n)``
+
         indices (list(int)): List of indices to be traced.
 
     Returns:
-        tensor_like: (reduced) Density matrix of size ``(batch_dim, 2**len(wires), 2**len(wires))``
+        tensor_like: (reduced) Density matrix of size ``(2**len(wires), 2**len(wires))``
 
     **Example**

```

Comment:
```suggestion
    **Example**
    
    We can compute the partial trace of the matrix ``x`` with respect to its first index.
```

Needs a line break as it currently does not render 
![image](https://github.com/PennyLaneAI/pennylane/assets/43949391/9d88b8e9-45a2-4872-86a4-cb74c84ccc7a)

Also add text to explain what is going on